modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCacheSettings.java

```diff
@@ -109,13 +111,15 @@ public class TieredSpilloverCacheSettings {
     );
 
     /**
-     * Setting defining the minimum took time for a query to be allowed into the disk cache.
+     * Setting defining the minimum took time for a query to be allowed in the cache.
+     * For backwards compatibility, the setting key still has "disk" in it, but the threshold is applied to enter into
+     * any tier of the cache.
      */
-    private static final Setting.AffixSetting<TimeValue> TIERED_SPILLOVER_DISK_TOOK_TIME_THRESHOLD = Setting.suffixKeySetting(
+    private static final Setting.AffixSetting<TimeValue> TIERED_SPILLOVER_TOOK_TIME_THRESHOLD = Setting.suffixKeySetting(
```

Comment:
I think it would be better to introduce a new setting for overall Tiered cache rather than reusing existing one. As it would be very confusing to users/anyone with the setting name which otherwise works on both onHeap/disk tier.

We could eventually decide to deprecate this(disk policy setting) altogether  later if needed.